apps/web/src/lib/types/property.ts

```
@@ -0,0 +1,103 @@
+  }
+  
+  export interface PropertyReviewsSectionProps {
+    propertyId: string
+    averageRating?: number
+    totalReviews?: number
+    className?: string
+  }
+  
+  export interface PropertyCalendarProps {
+    unavailableDates?: Date[]
+    onDateSelect?: (dateRange: any) => void
```

Biome (1.9.4)

[error] Unexpected any. Specify a different type.

any disables many type checking rules. Its use should be avoided.

(lint/suspicious/noExplicitAny)